siliconcompiler/use.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2023 Silicon Compiler Authors. All Rights Reserved.
+
+from siliconcompiler import Chip
+
+class PDK(Chip):
```

Comment:
I think it would be a good idea to make these `Schema` sub-classes, especially if we want them to eventually be persistent within the `Chip` object. But like we discussed, that can come in a future PR because of the complications with `Flow.node` / `Flow.edge`.